docs/troubleshooting/monitoring.md

```diff
@@ -69,3 +69,121 @@ rancher-monitoring-grafana-d9c56d79b-cp86w               3/3     Running   0
 
 ```
 
+## Expand PV/Volume Size
+
+`Harvester` integrates `Longhorn` as the default storage provider.
+
+Harvester `Monitoring` uses `Persistant Volume (PV)` to store running data. When a cluster has been running for a certain time, the `Persistant Volume` may need to expand its size.
+
```

Comment:
```suggestion
Harvester `Monitoring` uses `Persistent Volume (PV)` to store running data. When a cluster has been running for a certain time, the `Persistent Volume` may need to expand its size.

```

Reply:
I found that some previous suggesions haven't been applied but marked as resolved. Please double-check~